eva-server/src/main/java/uk/ac/ebi/eva/server/Utils.java

```diff
@@ -59,4 +78,19 @@ public static PageRequest getPageRequest(QueryOptions queryOptions) {
         return new PageRequest(page, size);
     }
 
+    public static String createExclusionFieldString(List<String> excludeList) {
+        List<String> formattedList = excludeList.stream().map(field -> String.format("'%s' : 0", field))
+                                                .collect(Collectors.toList());
+        return "{ " + String.join(", ", formattedList) + " }";
+    }
+
+    private static Map<String, String> initApiToMongoDocNameMap() {
+        Map<String, String> map = new HashMap<>();
+        map.put("sourceEntries", "files");
+        map.put("sourceEntries.statistics", "st");
+        map.put("annotation", "annot");
+        map.put("filesAttrs", "files.attrs");
```

Comment:
The key should be `sourceEntries.attributes` to match the names returned by the web service.